R/stat_stepribbon.R

```diff
@@ -0,0 +1,87 @@
+#' Step ribbon statistic
+#'
+#' Provides stair-step values for ribbon plots, often using in
+#' conjunction with `ggplot2::geom_step()`.
+#' The step ribbon can be added with `stat_stepribbon()` or
+#' identically with `ggplot2::geom_ribbon(stat = "stepribbon")`
```

Comment:
Thanks for the explanation but the user still does not know why he would choose this option instead of what the package provides by default. Could you add some rationale for this function?

Reply:
I'll open an issue that this needs to be expanded on